rs/types/types/src/consensus/idkg.rs

```diff
@@ -1560,86 +1554,8 @@ impl SignedBytesWithoutDomainSeparator for SignedIDkgOpening {
-            IDkgMessageAttribute::EcdsaSigShare(id) => Kind::EcdsaSigShare(id.into()),
-            IDkgMessageAttribute::SchnorrSigShare(id) => Kind::SchnorrSigShare(id.into()),
-            IDkgMessageAttribute::Complaint(id) => Kind::Complaint((&id).into()),
-            IDkgMessageAttribute::Opening(id) => Kind::Opening((&id).into()),
-        };
-        Self { kind: Some(kind) }
-    }
-}
-
-impl TryFrom<pb::IDkgMessageAttribute> for IDkgMessageAttribute {
-    type Error = ProxyDecodeError;
-    fn try_from(value: pb::IDkgMessageAttribute) -> Result<Self, Self::Error> {
-        use pb::i_dkg_message_attribute::Kind;
-        let Some(kind) = &value.kind else {
-            return Err(ProxyDecodeError::MissingField("IDkgMessageAttribute::kind"));
-        };
-        Ok(match &kind {
-            Kind::SignedDealing(id) => IDkgMessageAttribute::Dealing(id.try_into()?),
-            Kind::DealingSupport(id) => IDkgMessageAttribute::DealingSupport(id.try_into()?),
-            Kind::EcdsaSigShare(id) => IDkgMessageAttribute::EcdsaSigShare(id.try_into()?),
-            Kind::SchnorrSigShare(id) => IDkgMessageAttribute::SchnorrSigShare(id.try_into()?),
-            Kind::Complaint(id) => IDkgMessageAttribute::Complaint(id.try_into()?),
-            Kind::Opening(id) => IDkgMessageAttribute::Opening(id.try_into()?),
-        })
-    }
-}
-
-impl From<&IDkgMessage> for IDkgMessageAttribute {
-    fn from(msg: &IDkgMessage) -> IDkgMessageAttribute {
-        match msg {
-            IDkgMessage::Dealing(dealing) => {
-                IDkgMessageAttribute::Dealing(dealing.content.transcript_id)
-            }
-            IDkgMessage::DealingSupport(support) => {
-                IDkgMessageAttribute::DealingSupport(support.transcript_id)
-            }
-            IDkgMessage::EcdsaSigShare(share) => {
-                IDkgMessageAttribute::EcdsaSigShare(share.request_id.clone())
-            }
-            IDkgMessage::SchnorrSigShare(share) => {
-                IDkgMessageAttribute::SchnorrSigShare(share.request_id.clone())
-            }
-            IDkgMessage::Complaint(complaint) => {
-                IDkgMessageAttribute::Complaint(complaint.content.idkg_complaint.transcript_id)
-            }
-            IDkgMessage::Opening(opening) => {
-                IDkgMessageAttribute::Opening(opening.content.idkg_opening.transcript_id)
-            }
-        }
-    }
-}
-
-impl IDkgMessageAttribute {
-    pub fn as_str(&self) -> &'static str {
-        match self {
-            Self::Dealing(_) => "signed_dealing",
-            Self::DealingSupport(_) => "dealing_support",
-            Self::EcdsaSigShare(_) => "ecdsa_sig_share",
-            Self::SchnorrSigShare(_) => "schnorr_sig_share",
-            Self::Complaint(_) => "complaint",
-            Self::Opening(_) => "opening",
-        }
-    }
-}
+/// The final output of the transcript creation sequence
+pub type EcdsaTranscript = IDkgTranscript;
```

Comment:
I think this might have been added back in accidentally during a merge
```suggestion
```

Reply:
sorry about that!